models/users.models.js

```diff
@@ -0,0 +1,9 @@
+const db = require("./../db/connection.js");
+
+exports.fetchUsers = () => {
+    return db.query("SELECT * FROM users").then(({rows}) => {
+        return rows.map((user) => {
```

Comment:
return the whole user object, you don't want to be filtering out information here